pkg/sql/rowexec/inverted_joiner.go

```diff
@@ -41,7 +41,10 @@ import (
 // higher scan throughput of larger batches and the cost of spilling the
 // scanned rows to disk. The spilling cost will probably be dominated by
 // the de-duping cost, since it incurs a read.
-const invertedJoinerBatchSize = 100
+var invertedJoinerBatchSize = util.ConstantWithTestValue(
+	100, /* production */
+	1,   /* test */
```

Comment:
[nit] `production` and `test` are a bit confusing to me because when I see these I assume all tests run with the `test` constant, and `production` is only used for release binaries. But really its more like a `default` which is the normal value and some special value (name brainstorm: `alternate`, `substitute`, or leave as `test`???).

Reply:
I've renamed the build tag, constant and methods to be about "metamorphic testing".